boards/arm/raytac_mdbt50q_db_33_nrf52833/Kconfig

```diff
@@ -0,0 +1,10 @@
+# Raytac MDBT50Q-DB-33 nRF52833 board configuration
```

Comment:
Change both commit messages to be short, e.g.
```
boards: arm: add raytac_mdbt50q_db_33_nrf52833

Adds new raytac_mdbt50q_db_33_nrf52833 board.

Signed off by...
```
i.e. sign off the commits and remove the list of features in the board, they do not go in a commit message for a board